### ISSUE TITLE
[Bug]:Warning message in log ( defaultDaemonConfig) when navigating to "Applications" tab and submenu, Nextcloud 30.0.2 RC2

### ISSUE THREAD

**Issue by Cisco30** [nextcloud/server#49123]
### ⚠️ This issue respects the following points: ⚠️

- [x] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [x] This issue is **not** already reported on [Github](https://github.com/nextcloud/server/issues?q=is%3Aopen+is%3Aissue+label%3Abug) OR [Nextcloud Community Forum](https://help.nextcloud.com/) _(I've searched it)_.
- [x] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [x] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

Hi, A warning message is generated each time when browsing the "Applications" tab. This message is generated each time in the submenus of the "Applications" tab, and each time the web page is refreshed inside the tab.

![Image](https://github.com/user-attachments/assets/0a19f005-20c5-49ab-bc59-e01a293dbd64)

### Steps to reproduce

1.Go to the “Applications” tab
2.check the Nextcloud error log
3.each time the tab is refreshed the error is added

### Expected behavior

there should be no errors in the log when navigating to the "Applications" tab

### Nextcloud Server version

30

### Operating system

Debian/Ubuntu

### PHP engine version

PHP 8.2

### Web server

Apache (supported)

### Database engine version

MariaDB

### Is this bug present after an update or on a fresh install?

Updated from a MINOR version (ex. 28.0.1 to 28.0.2)

### Are you using the Nextcloud Server Encryption module?

None

### What user-backends are you using?

- [x] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Configuration report

    "system": {
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "cloud.seeboxs.com",
            "www.cloud.seeboxs.com"
        ],
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "mysql",
        "version": "30.0.2.1",
        "overwrite.cli.url": "https:\/\/cloud.seeboxs.com",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "oc_",
        "mysql.utf8mb4": true,
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "default_phone_region": "FR",
        "htaccess.RewriteBase": "\/",
        "trashbin_retention_obligation": "auto, 30",
        "filelocking.enabled": "true",
        "knowledgebaseenabled": false,
        "activity_expire_days": 7,
        "skeletondirectory": "\/var\/www\/extra_nextcloud\/skeleton\/",
        "redis": {
            "host": "***REMOVED SENSITIVE VALUE***",
            "port": "0",
            "timeout": "0.0"
        },
        "memcache.distributed": "\\OC\\Memcache\\Redis",
        "memcache.local": "\\OC\\Memcache\\Redis",
        "memcache.locking": "\\OC\\Memcache\\Redis",
        "mail_smtpmode": "smtp",
        "mail_smtpsecure": "ssl",
        "mail_sendmailmode": "smtp",
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpport": "465",
        "mail_smtpauth": 1,
        "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
        "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
        "default_language": "fr",
        "default_locale": "fr_FR",
        "updater.release.channel": "stable",
        "maintenance_window_start": 1,
        "maintenance": false,
        "integrity.check.disabled": false,
        "enable_previews": true,
        "enabledPreviewProviders": {
            "1": "OC\\Preview\\BMP",
            "2": "OC\\Preview\\GIF",
            "3": "OC\\Preview\\JPEG",
            "4": "OC\\Preview\\Krita",
            "5": "OC\\Preview\\MarkDown",
            "6": "OC\\Preview\\MP3",
            "7": "OC\\Preview\\OpenDocument",
            "8": "OC\\Preview\\PNG",
            "9": "OC\\Preview\\TXT",
            "10": "OC\\Preview\\XBitmap",
            "11": "OC\\Preview\\Image",
            "12": "OC\\Preview\\HEIC",
            "13": "OC\\Preview\\TIFF",
            "14": "OC\\Preview\\Movie",
            "15": "OC\\Preview\\MP4",
            "16": "OC\\Preview\\PDF"
        },
        "preview_max_x": 1024,
        "preview_max_y": 1024,
        "preview_concurrency_all": 16,
        "preview_concurrency_new": 8,
        "preview_max_filesize_image": 200,
        "preview_max_memory": -1,
        "preview_ffmpeg_path": "\/usr\/bin\/ffmpeg",
        "theme": "",
        "defaultapp": "",
        "loglevel": 2,
        "updatechecker": true,
        "has_internet_connection": true,
        "app_install_overwrite": [
            "twofactor_email",
            "otpmanager",
            "socialsharing_facebook",
            "maps"
        ],
        "forbidden_filename_characters": [
            "\\",
            "\/"
        ],
        "forbidden_filename_extensions": [
            ".filepart",
            ".part"

### List of activated Apps

Enabled:
  - activity: 3.0.0
  - admin_audit: 1.20.0
  - announcementcenter: 7.0.1
  - app_api: 4.0.0
  - bruteforcesettings: 3.0.0
  - calendar: 5.0.1
  - cloud_federation_api: 1.13.0
  - comments: 1.20.1
  - contacts: 6.1.0
  - contactsinteraction: 1.11.0
  - dashboard: 7.10.0
  - dav: 1.31.1
  - drop_account: 2.6.1
  - external: 5.5.2
  - federatedfilesharing: 1.20.0
  - federation: 1.20.0
  - files: 2.2.0
  - files_accesscontrol: 1.20.1
  - files_downloadlimit: 3.0.0
  - files_external: 1.22.0
  - files_pdfviewer: 3.0.0
  - files_reminders: 1.3.0
  - files_sharing: 1.22.0
  - files_trashbin: 1.20.1
  - files_versions: 1.23.0
  - firstrunwizard: 3.0.0
  - guests: 4.0.1
  - logreader: 3.0.0
  - lookup_server_connector: 1.18.0
  - mail: 4.0.2
  - maps: 1.5.0
  - metadata: 0.21.0
  - nextcloud_announcements: 2.0.0
  - notes: 4.11.0
  - notifications: 3.0.0
  - oauth2: 1.18.1
  - otpmanager: 0.5.4
  - passman: 2.4.11
  - password_policy: 2.0.0
  - photos: 3.0.2
  - previewgenerator: 5.6.0
  - provisioning_api: 1.20.0
  - quota_warning: 1.20.0
  - registration: 2.5.0
  - related_resources: 1.5.0
  - richdocuments: 8.5.2
  - serverinfo: 2.0.0
  - settings: 1.13.0
  - sharebymail: 1.20.0
  - socialsharing_diaspora: 3.2.0
  - socialsharing_email: 3.2.0
  - socialsharing_facebook: 3.2.0
  - socialsharing_twitter: 3.2.0
  - spreed: 20.0.1
  - suspicious_login: 8.0.0
  - systemtags: 1.20.0
  - tasks: 0.16.1
  - terms_of_service: 4.0.0
  - text: 4.1.0
  - theming: 2.5.0
  - theming_customcss: 1.17.0
  - twofactor_backupcodes: 1.19.0
  - twofactor_email: 2.7.4
  - twofactor_nextcloud_notification: 4.0.0
  - twofactor_totp: 12.0.0-dev
  - unroundedcorners: 1.1.4
  - updatenotification: 1.20.0
  - user_retention: 1.13.0
  - user_status: 1.10.0
  - viewer: 3.0.0
  - weather_status: 1.10.0
  - webhook_listeners: 1.1.0-dev
  - workflowengine: 2.12.0
Disabled:
  - circles: 30.0.0 (installed 27.0.1)
  - encryption: 2.18.0
  - privacy: 2.0.0 (installed 1.12.0)
  - recommendations: 3.0.0 (installed 1.6.0)
  - support: 2.0.0 (installed 1.11.1)
  - survey_client: 2.0.0 (installed 2.0.0)
  - user_ldap: 1.21.0 (installed 1.19.0)

### Nextcloud Signing status

No errors have been found.

### Nextcloud Logs

```
{"reqId":"Ckoa6YfQRfpgWbHlXkuJ","level":2,"time":"2024-11-07T08:16:46+00:00","remoteAddr":"192.168.1.254","user":"Boubou","app":"no app in context","method":"GET","url":"/settings/apps","message":"Invalid defaultDaemonConfig data provided to provideInitialState by settings","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/130.0.0.0 Safari/537.36","version":"30.0.2.1","data":[]}
```

### Additional info

I am using:
Debian 12 (up to date)
MariaDB Server version: 11.4.4-MariaDB-deb12
PHP 8.2.24 (cli)
Apache/2.4.62 (Debian)

**Comment by Scoon1e** [comment]
I get the same error without the windows compatibility.
For me it is the plugin "Passwords". After Updating from 30.0.1 to 30.0.2 I was not able to access Passwords nor the administrative settings via Web-GUI.

After deactivating the plugin Passwords I have access again and can read the thrown error I have attached:

![Image](https://github.com/user-attachments/assets/397f3c83-c64d-4de2-9dd7-aa36df584f7e)

**Comment by leachimus** [comment]
The app “app_api: 4.0.0” is responsible for this behavior. I have just deactivated all my apps and activated them individually, and this app is the culprit. After deactivation there are no more errors.
I can't understand why this wasn't noticed before the release.

**Comment by vanyolai** [comment]
> I get the same error without setting windows compatibility. For me it is the plugin "Passwords" what leads to this error. After Updating from 30.0.1 to 30.0.2 I was not able to access Passwords nor the administrative settings via Web-GUI.
> 
> After deactivating the plugin Passwords I have access again and can read the thrown error I have attached:
> 
> ![Image](https://github.com/user-attachments/assets/397f3c83-c64d-4de2-9dd7-aa36df584f7e)

I've got the same issue. I had to disable passwords app, until then I kept getting internal server error and could not open dashboard.

**Comment by Pammetje** [comment]
> The app “app_api: 4.0.0” is responsible for this behavior. I have just deactivated all my apps and activated them individually, and this app is the culprit. After deactivation there are no more errors. I can't understand why this wasn't noticed before the release.

Also disabled app_api: 4.0.0” and the warnings are also gone on my system

**Comment by Scoon1e** [comment]
I've got an update for the app Passwords today.
Now everything is working again. No need to deactivate the "app_api: 4.0.0".

![Image](https://github.com/user-attachments/assets/36eaaca3-b899-4749-a12a-f80d89426c07)

**Comment by HRL62** [comment]
> > The app “app_api: 4.0.0” is responsible for this behavior. I have just deactivated all my apps and activated them individually, and this app is the culprit. After deactivation there are no more errors. I can't understand why this wasn't noticed before the release.
> 
> Also disabled app_api: 4.0.0” and the warnings are also gone on my system

**Comment by csterling8** [comment]
I have the same issue and I do not have the passwords app installed. Will try disabling app_api 4.0.0 . Other than the warnings everything seems to be working ok.

**Comment by nigelharpur** [comment]
I just disabled app_api: 4.0.0 after reading @leachimus comment and the log error has stopped immediately - as far as I can tell I currently have no apps that need app_api: 4.0.0 on my system so for now 'a fix' but obviously still needs fixing at source.

**Comment by Gekko23** [comment]
The problem hasn't been touched yet, it passed Quality Mangement even for the newest update on 30.0.3
I don't understand how this can pass through every instance of development without being noticed. 
"Fixed" this by deactivating the app_api 4.0.0 inside 30.0.3

**Comment by ozgurkazancci** [comment]
Got the very same issue under Nextcloud 30.0.4, always filling Nextcloud log with such warning;

```
{
  "reqId": "WARRV71KCzCb08PKkEY4",
  "level": 2,
  "time": "2024-12-17T06:03:38+00:00",
  "remoteAddr": "xx.xx.xx",
  "user": "admin",
  "app": "no app in context",
  "method": "GET",
  "url": "/settings/apps",
  "message": "Invalid defaultDaemonConfig data provided to provideInitialState by settings",
  "userAgent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36",
  "version": "30.0.4.1",
  "data": [],
  "id": "67611464470d8"
}
```

**Comment by phsc84** [comment]
At least for me it had no side effects. Because I also disabled AppAPI due to this bug.

**Comment by axheli** [comment]
Also seeing this.

```
{"reqId":"jQOb2ExdWDMRu7XBnKF6","level":2,"time":"2024-12-30T08:20:54-06:00","remoteAddr":"1.1.1.1","user":"admin","app":"no app in context","method":"GET","url":"/settings/apps","message":"Invalid defaultDaemonConfig data provided to provideInitialState by settings","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36","version":"30.0.4.1","data":[],"id":"6772ac50dcd67"}

```

**Comment by bigretromike** [comment]
AppApi is throwing this. disable until fix

**Comment by Cisco30** [comment]
hi
just noticed that with version 30.0.5 Debian 12, the problem was solved, when navigating in the Applications menu there is no more error in the Nextcloud log

**Comment by nigelharpur** [comment]
Is this with the APIapp re-enabled?

On 6 February 2025 12:19:22 GMT, XYZ ***@***.***> wrote:
>Do I close the issue?
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/nextcloud/server/issues/49123#issuecomment-2639672032
>You are receiving this because you commented.
>
>Message ID: ***@***.***>
Live Long and Prosper

**Comment by DerDreschner** [comment]
> Is this with the APIapp re-enabled?
Yeah, I re-enabled it in the meantime and dealt with the error messages.

**Comment by nigelharpur** [comment]
Yep, can confirm here, re-enabled appAPI, went in and out of Apps screen a few times with no errors in log. Logged out and back in, repeated tests and still no errors logged.

Seems fixed for me.

On 6 February 2025 12:52:06 GMT, David Dreschner ***@***.***> wrote:
>> Is this with the APIapp re-enabled?
>
>Yeah, I re-enabled it in the meantime and dealt with the error messages.
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/nextcloud/server/issues/49123#issuecomment-2639744964
>You are receiving this because you commented.
>
>Message ID: ***@***.***>
Live Long and Prosper

**Comment by Cisco30** [comment]
Hi
Can I close this? I no longer see any problems with version 31.0.2.1.

**Comment by nigelharpur** [comment]
No issues for me for over a month. Currently on 30.0.8 so all fixed from my pov

On 23 March 2025 08:18:32 GMT, XYZ ***@***.***> wrote:
>Cisco30 left a comment (nextcloud/server#49123)
>
>Hi
>Can I close this? I no longer see any problems with version 31.0.2.1.
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/nextcloud/server/issues/49123#issuecomment-2746085503
>You are receiving this because you commented.
>
>Message ID: ***@***.***>
Live Long and Prosper
